contracts/vault/packages/VaultPackage.sol

```diff
@@ -645,7 +657,7 @@ contract VaultPackage is VaultStorage, IVault, IVaultInit, IVaultEvents {
     /// @return The maximum amount of shares that can be redeemed.
     function maxRedeem(address owner, uint256 maxLoss, address[] calldata _strategies) external view override returns (uint256) {
         uint256 maxWithdrawAmount = _maxWithdraw(owner, maxLoss, _strategies);
-        uint256 sharesEquivalent = _convertToShares(maxWithdrawAmount, Rounding.ROUND_UP);
+        uint256 sharesEquivalent = _convertToShares(maxWithdrawAmount, Rounding.ROUND_DOWN);
```

Comment:
Is it related to issue in _assessShareOfUnrealisedLosses?

Reply:
No, this one is related to Warning 04 about maxRedeem rounding. Another rounding issue. But this one I think we shall keep the fix after reviewing, still makes sense to me.

Reply:
No, this one is related to Warning 04 about maxRedeem rounding. Another rounding issue. But this one I think we shall keep the fix after reviewing, still makes sense to me.